pkg/utils/k8sutil.go

```diff
@@ -611,7 +612,7 @@ func DeployKubeless(client *kubernetes.Clientset, ctlImage string, ctlNamespace
 // DeployMsgBroker deploys kafka-controller
 func DeployMsgBroker(client *kubernetes.Clientset, kafkaVer string, ctlNamespace string) error {
 	labels := map[string]string{
-		"app": "kafka",
+		"function": "kafka-controller",
```

Comment:
Yes it was only for reusing `GetPods` with the kafka case. Agree with changing to `GetPodByLabel` function which can be generic to solve both function and non-function cases.